### PR TITLE
Added Centered property to ISprites

### PR DESCRIPTION
**Pull request by zorg93**
Setting to false: position specified corresponds to the top left of the sprite(Like all other UI.Elements)
Setting to true: position specified corresponds to the center of the sprite

### CODE REVIEW

**Comment by Fireboyd78** [comment]
Nice!

File: source/scripting/Vehicle.hpp [hunk 1]
```diff
@@ -311,12 +311,13 @@ namespace GTA
 	};
 	public enum class VehicleWindowTint
 	{
+		GrangerNone = -1,
```
Comment (Fireboyd78) [inline source/scripting/Vehicle.hpp]: What's `GrangerNone`? Would it possibly be tinted rear windows only?

Comment (zorg93) [inline source/scripting/Vehicle.hpp]: A special tint only available on the Granger, I found it looking in LSC script

Comment (Fireboyd78) [inline source/scripting/Vehicle.hpp]: I know, but doesn't the Granger have tinted rear windows and clear front windows?

**Comment by crosire** [comment]
The UI changes were merged as part of #490. The changes to ```VehicleWindowTint``` however cannot be merged with this pull request due to the C# restructure and need to be addressed in a follow-up.

**Pull request closed without merging**